### PR TITLE
[WIP] AEIM-2663 quod dev

### PR DESCRIPTION
**Pull request by sulemandiwan**
Initial quod-dev profile for tang & co. and some quod ground work.

### CODE REVIEW

File: manifests/profile/quod/perl.pp [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    'libxml-simple-perl',
+    'libxml-twig-perl',
+    'libxml-writer-perl',
+    'libxml-xpath-perl',
+    'libxml-xpathengine-perl',
+    'libyaml-appconfig-perl',
+    'libyaml-libyaml-perl',
+    'libyaml-perl',
+    'libyaml-syck-perl',
+    'xmlformat-perl']:
```
Comment (daaang) [inline manifests/profile/quod/perl.pp]: Where did this list come from? Why is it different from "dependencies"?

File: manifests/profile/quod/perl.pp [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+  }
+
+  -> nebula::cpan { [
```
Comment (daaang) [inline manifests/profile/quod/perl.pp]: Do these 13 nebula::cpan packages really depend on all 245 system packages?

File: manifests/profile/quod/dependencies.pp [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
```
Comment (daaang) [inline manifests/profile/quod/dependencies.pp]: 2020

File: manifests/profile/quod/perl.pp [hunk 4]
```diff
@@ -0,0 +1,284 @@
+
+# Copyright (c) 2019 The Regents of the University of Michigan.
```
Comment (daaang) [inline manifests/profile/quod/perl.pp]: 2020

File: manifests/role/app_host/quod_dev.pp [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
```
Comment (daaang) [inline manifests/role/app_host/quod_dev.pp]: 2020

File: manifests/profile/quod/perl.pp (hunk 2 above)
Comment (sulemandiwan) [reply manifests/profile/quod/perl.pp]: They are based off of what is on the machine. And we don't know the dependency chains.

File: manifests/profile/quod/perl.pp (hunk 1 above)
Comment (sulemandiwan) [reply manifests/profile/quod/perl.pp]: These are perl packages. Dependencies are prerequisites for these packages as well as for anything else that needs to be installed.

The layout is based on what we have in www_lib.

Comment (sulemandiwan) [reply manifests/profile/quod/perl.pp]: And example would be the oracle auth which apparently is a pre-requisite for some perl packages from what I gathered.

**Pull request merged**